src/main/java/pl/betoncraft/betonquest/compatibility/citizens/CitizensListener.java

```diff
@@ -31,40 +31,41 @@
 
 /**
  * Starts new conversations with NPCs
- * 
+ *
  * @author Jakub Sapalski
  */
 public class CitizensListener implements Listener {
 
-	/**
-	 * Initializes the listener
-	 */
-	public CitizensListener() {
-		Bukkit.getPluginManager().registerEvents(this, BetonQuest.getInstance());
-	}
+    /**
+     * Initializes the listener
+     */
+    public CitizensListener() {
+        Bukkit.getPluginManager().registerEvents(this, BetonQuest.getInstance());
+    }
 
-	@EventHandler(priority = EventPriority.HIGHEST)
-	public void onNPCClick(final NPCRightClickEvent event) {
-		if (event.isCancelled()) {
-			return;
-		}
-		if (!event.getClicker().hasPermission("betonquest.conversation")) {
-			return;
-		}
-		if (NPCMoveEvent.isNPCMoving(event.getNPC())) {
-			return;
-		}
-		final String playerID = PlayerConverter.getID(event.getClicker());
-		if (CombatTagger.isTagged(playerID)) {
-			Config.sendMessage(playerID, "busy");
-			return;
-		}
-		String id = String.valueOf(event.getNPC().getId());
-		String assignment = Config.getNpc(id);
-		if (assignment != null) {
-			event.setCancelled(true);
-			new CitizensConversation(playerID, assignment, event.getNPC().getEntity().getLocation(),
-					event.getNPC());
-		}
-	}
+    @EventHandler(priority = EventPriority.HIGHEST)
+    public void onNPCClick(final NPCRightClickEvent event) {
+        if (event.isCancelled()) {
+            return;
+        }
+        if (!event.getClicker().hasPermission("betonquest.conversation")) {
+            return;
+        }
+        if (NPCMoveEvent.isNPCMoving(event.getNPC())) {
+            return;
+        }
+        final String playerID = PlayerConverter.getID(event.getClicker());
+        if (CombatTagger.isTagged(playerID)) {
+            Config.sendMessage(playerID, "busy");
+            return;
+        }
+        String id = String.valueOf(event.getNPC().getId());
+        String assignment = Config.getNpc(id);
+        if (assignment == null) assignment = Config.getNpc(event.getNPC().getName());
```

Comment:
I'dd suggest adding a config option which enables this feature which is false by default.
Otherwise this could screw up npcs on servers which use both, the npcs build out of clay and the citizens npcs.